src/index.js

```diff
@@ -47,7 +47,7 @@ class Network {
 
     on(event, callback) {
         if (typeof event !== 'string' || !NetworkStatuses.includes(event)) {
-            throw new Error(`Event must be one of ${NetworkStatuses}`)
+            throw new Error(`${ErrorMessage.INVALID_EVENT}`)
```

Comment:
Don't need the interpolation
```suggestion
            throw new Error(ErrorMessage.INVALID_EVENT)
```

Reply:
> This conversation was marked as resolved by StealthGiraffe

😢